qutebrowser/config/configdata.yml

```
@@ -1811,6 +1811,23 @@ input.match_counts:
     Disabling it allows for emacs-like bindings where number keys are passed
     through (according to `input.forward_unbound_keys`) instead.
 
+input.mode_override:
+  default: null
+  type:
+    name: String
+    none_ok: true
+    valid_values:
+      - normal
+      - insert
+      - hint
+      - passthrough
+      - command
+      - caret
+  supports_pattern: true
+  desc: Automatically change to the given mode when focusing on a tab.
```

Could you rephrase this to start either with noun referring to what is selected by this option or an interrogative word?
This would make it more consistent with the rest of the config data descriptions.

```suggestion
  desc: Mode to change to when focusing on a tab.
```

Personally I'm leaning towards leaving out the `which` in this case because `null` is a valid option and thus there is not necessarily a mode to be choosen.

---

Sounds good thanks!